syntax = "proto2";

message Workspace{
    required string path = 1;
    required string machineID = 2;
    required bool restore  = 3;
}

